kb/2023-02-01/scan_and_repair_filesystem.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Scan and Repair Root Filesystem of VirtualMachine
+description: Scan and repair root filesystem of VM
+slug: scan-and-repair-vm-root-filesystem
+authors:
+  - name: Vicente Cheng
+    title: Senior Software Engineer
+    url: https://github.com/Vicente-Cheng
+    image_url: https://github.com/Vicente-Cheng.png
+tags: [storage, longhorn, root, filesystem]
+hide_table_of_contents: false
+---
+
+In earlier versions of Harvester (v1.0.3 and prior), Longhorn volumes sometimes get corrupted when rebuilding (reference: [Analysis: Potential Data/Filesystem Corruption](https://longhorn.io/kb/troubleshooting-volume-filesystem-corruption/#solution)). In Harvester v1.1.0 and later versions, the Longhorn team has fixed this problem. So, you can follow the steps below to scan the VM's filesystem and repair it if needed.
```

Comment:
```suggestion
In earlier versions of Harvester (v1.0.3 and prior), Longhorn volumes sometimes get corrupted when rebuilding (reference: [Analysis: Potential Data/Filesystem Corruption](https://longhorn.io/kb/troubleshooting-volume-filesystem-corruption/#solution)). In Harvester v1.1.0 and later versions, the Longhorn team has fixed this problem. This article covers the steps you can take to scan the VM's filesystem and repair it if needed.
```

Reply:
Hi @LucasSaintarbor, the first half of the sentences changed the original meaning.
I would like to present that the old version longhorn sometime makes replicas corrupted when rebuilding.
How would we improve the sentences here?

Reply:
Hi @Vicente-Cheng, thanks for the feedback, is the first sentence correct? Does it summarize the issue?